src/app.js

```diff
@@ -1,21 +1,33 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+
+  const pureFile = normalizedURL.pathname.slice(6) || 'index.html';
+
+  if (!normalizedURL.pathname.startsWith('/file/')) {
+    res.statusCode = 404;
+
+    res.end(
+      'Please, use /file/ for the correct file path.'
+    );
```

Comment:
can be less
check other places 
```suggestion
    res.status(404).end(
      'Please, use /file/ for the correct file path.'
    );
```